### DISCUSSION TITLE
PDF Reports and Thai Tone Marks rendering

### DISCUSSION THREAD

**Discussion by gaizers** [eclipse-birt/birt#1133] [Q&A]
we have a issue when the report PDF renders Thai tone mark characters (as show in the image)
usually there should be 2 layers, but both tone marks are rendered on one layer.

we render with: birt-runtime-4_4_2

![1671186129096](https://user-images.githubusercontent.com/1132742/208148274-275db0d0-39fd-4d59-91a6-1101b6bf749d.jpg)
![1671209680706](https://user-images.githubusercontent.com/1132742/208149285-c3530b68-39d7-40a8-9da8-9f3d23cc01c2.jpg)

**Comment by wimjongman** [comment]
Please try with the latest BIRT runtime 4.12

https://eclipse.org/birt

**Reply by gaizers** [reply]
try with BIRT runtime 4.12, result are same issue.

**Reply by Rukyee** [reply]
I have same your problem.
Can you solve this problem ?

**Comment by hvbtup** [comment]
AFAIK all active committers use more or less latin character sets, none of them has an understanding of the intricacies of rendering text in asian languages. So probably it would be best if you dig in the source code yourself and determine what's wrong there.
